pkg/cmd/tenant/addrepo/tenant_add_repo.go

```diff
@@ -43,8 +45,15 @@ func NewTenantAddRepoCmd(cfg *config.Config) *cobra.Command {
 }
 
 func run(opts *TenantAddRepoOpts, cfg *config.Config) error {
+	opts.Streams.Wizard(
+		fmt.Sprintf("Adding repository to tenant: %s", opts.TenantName),
```

Comment:
is it owrth to add the link to tenancy repo?

Reply:
Added